apiary.apib

```diff
@@ -429,7 +429,7 @@ We list all backwards-incompatible changes here. As described above, new additio
 
 #### 2019-07-03
 
-- time added to work orders now has multiple users instead of one. You can pass a list of uuid's in the workOrders.draft and workOrders.update endpoints and will receive a list of user entities in workOrders.info.
+- Time added to work orders now has multiple users instead of one. You can pass a list of uuids in the workOrders.draft and workOrders.update endpoints and will receive a list of user entities in workOrders.info.
```

Comment:
B: "uuids" should be "UUIDs" in my opinion. Also the source file was not updated here.